x-pack/plugins/security_solution/server/utils/custom_request_error.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+export class CustomRequestError extends Error {
```

Comment:
(optional) Because the error is some what targeted to an HTTP response, should its name instead be: `CustomHttpResponseError`?

Reply:
Indeed, that is a more sensible name.

Reply:
522681f0759b28b817c0ea5a8aa01b9aead0efb6